src/Makefile.am

```diff
@@ -91,6 +83,12 @@ $(LIBUNIVALUE): $(wildcard univalue/lib/*)
 $(LIBCRYPTOCONDITIONS): $(wildcard cryptoconditions/src/*) $(wildcard cryptoconditions/include/*)
 	$(AM_V_at)$(MAKE) $(AM_MAKEFLAGS) -C $(@D) $(@F) OPTFLAGS="-O2 -march=x86-64 -g "
 
+%.o: %.c
+	$(CC) -c -o $@ $<
+
+$(LIBCJSON): cJSON.o komodo_cJSON.o komodo_cutils.o
+	$(AR) cr $(LIBCJSON) $^
+
```

Comment:
btw, about the way how `libcjson.a` builds, may be the following approach is more clearly:
https://github.com/DeckerSU/KomodoOcean/blob/5f99d88568e0a49ce654135d59cb8e191f5667cd/src/Makefile.am#L77-L82
```
# libcjson build
LIBCJSON=libcjson.a
libcjson_a_SOURCES = cJSON.c \
  komodo_cJSON.c \
  komodo_cutils.cpp
libcjson_a_CPPFLAGS=-fPIC
EXTRA_LIBRARIES += $(LIBCJSON)
```
it's just like "open discussion" how it would better.

**p.s.** other tests of this PR still continues on my side.

Reply:
Also need to add clean of `libcjson.a` on `make clean`, like this:
https://github.com/DeckerSU/KomodoOcean/commit/53ddb012484223747e1ef18140864209d32e8311